src/components/TitleBody/TitleBody.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+export function TitleBody({
+  prefix,
+  title,
+  body,
+}: {
+  prefix?: React.ReactNode;
+  title: React.ReactNode;
+  body: React.ReactNode;
+}) {
+  return (
+    <div>
+      {prefix}
+      <strong style={{ fontSize: 14 }}>{title}</strong>
```

Comment:
let's already use styled components to make it easier later to adjust styling. let's not use inline styles.